app/models/manageiq/providers/vmware/cloud_manager.rb

```diff
@@ -2,6 +2,7 @@ class ManageIQ::Providers::Vmware::CloudManager < ManageIQ::Providers::CloudMana
   require_nested :RefreshParser
   require_nested :RefreshWorker
   require_nested :Refresher
+  require_nested :Template
```

Comment:
@gberginc looks like they are for the service broker
@chriskacerguis @jntullo should new providers be adding a `VirtualTemplate` class?